src/main/java/ChatResponseRespone.java

```diff
@@ -0,0 +1,29 @@
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ChatResponseRespone {
+    private List<Choice> choices;
```

Comment:
The class name 'ChatResponseRespone' seems to have a typo. It should be 'ChatResponseResponse' for better readability and adherence to naming conventions.